src/components/organization-chart/organization-chart.component.ts

```diff
@@ -384,6 +387,10 @@ export class OrganizationChartComponent<T> implements AfterViewInit, OnChanges,
     /** Toggle the collapsed state of a node */
     toggle(node: OrganizationChartNode<T> | HierarchyPointNode<OrganizationChartNode<T>>): void {
 
+        if (!this.allowToggling) {
```

Comment:
I would suggest moving this inside the if statement below it which also immediately returns. E.g. `if (this._isTransitioning || !this.allowToggling)` and you can then remove the comment above that since it is no longer accurate